apps/login/src/app/(login)/signedin/page.tsx

```diff
@@ -1,20 +1,30 @@
 import { createCallback, getBrandingSettings, getSession } from "@/lib/zitadel";
 import DynamicTheme from "@/ui/DynamicTheme";
 import UserAvatar from "@/ui/UserAvatar";
+import { createMessage } from "@zitadel/client";
 import { getMostRecentCookieWithLoginname } from "@zitadel/next";
 import { redirect } from "next/navigation";
+import {
+  CreateCallbackRequestSchema,
+  SessionSchema,
+} from "@zitadel/proto/zitadel/oidc/v2/oidc_service_pb";
 
 async function loadSession(loginName: string, authRequestId?: string) {
   const recent = await getMostRecentCookieWithLoginname({ loginName });
 
   if (authRequestId) {
-    return createCallback({
-      authRequestId,
-      callbackKind: {
-        case: "session",
-        value: { sessionId: recent.id, sessionToken: recent.token },
-      },
-    }).then(({ callbackUrl }) => {
+    return createCallback(
+      createMessage(CreateCallbackRequestSchema, {
```

Comment:
**Important:** Why do I have to call `createMessage`? The type system will complain because `$typeName` is not defined; the intent is that you go thru the `createMessage` factory function to guarantee correctness